airflow/config_templates/config.yml

```diff
@@ -1090,7 +1090,7 @@ metrics:
       version_added: 2.6.0
       type: string
       example: "\"scheduler,executor,dagrun,pool,triggerer,celery\"
-      or \"^scheduler,^executor,heartbeat|timeout\""
+        or \"^scheduler,^executor,heartbeat|timeout\""
```

Comment:
Probably should use `|` string instead?

Reply:
In here I just fixed the indent which was leading to "bad" YAML, content-wise did not want to touch it here.